moderncv.cls

```diff
@@ -276,6 +304,15 @@
       \ifthenelse{\equal{#1}{googlescholar}}{\collectionadd[googlescholar]{socials}{\protect\httpslink[#3]{scholar.google.com/citations?user=#3}}} {}%
       \ifthenelse{\equal{#1}{codeberg}}     {\collectionadd[codeberg]{socials}     {\protect\httpslink[#3]{codeberg.org/#3}}}                      {}%
       \ifthenelse{\equal{#1}{discord}}      {\collectionadd[discord]{socials}      {#3}}                                                           {}%
+      \ifthenelse{\equal{#1}{twitch}}       {\collectionadd[twitch]{socials}       {\protect\httpslink[#3]{twitch.tv/#3}}}                         {}%
+      \ifthenelse{\equal{#1}{youtube}}      {\collectionadd[youtube]{socials}      {\protect\httpslink[#3]{youtube.com/#3}}}                       {}%
```

Comment:
I don't think the address is correct here.

Reply:
As it turns out, YouTube has several address formats depending on how and when the YouTube account was set up:
- User (old accounts) `https://www.youtube.com/Quaddicted` (or `https://www.youtube.com/user/Quaddicted`)
- Channel URL `https://www.youtube.com/channel/UCMb0O2CdPBNi-QqPk5T3gsQ`
- Custom URL `https://www.youtube.com/c/StrangeParts`

https://support.google.com/youtube/answer/6180214?hl=en-GB 

Given that, I think it's acceptable for the rendered ID to be either `username` or `c/channelname` to accommodate, but open to suggestions.

Reply:
Maybe add one or two examples in the template to show how to add: there's already the username one. Maybe specify it can have c/ and user/ in front, and then show a second example with channel/id ?

Reply:
Done!

```tex
% YouTube links can take several forms, depending on how your account and channel are set up.
% See https://support.google.com/youtube/answer/6180214 for more information.
\social[youtube]{c/jdoeschannel}                   % optional, remove / comment the line if not wanted; Custom URL             - can be shorted by removing 'c/'
\social[youtube]{channel/XXXXXX}                   % optional, remove / comment the line if not wanted; Channel URL (ID-based) - can not be shortened
\social[youtube]{user/jdoe}                        % optional, remove / comment the line if not wanted; Legacy username URL    - if jdoe is not already claimed by a Custom URL, can be shortened by removing 'user/'
```

![Screenshot_20220824_154245](https://user-images.githubusercontent.com/1525809/186453325-446a928e-3ca6-4d53-882f-1714f1b5d477.png)